webdriver/tests/execute_async_script/pinning.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from webdriver.transport import Response
+
+from tests.support.asserts import assert_error, assert_success
+
+
+def pin_script(session, name, script):
+    body = {"script": script}
+
+    return session.transport.send(
+        "POST", "/session/{session_id}/execute/pin/{script_name}".format(
```

Comment:
There’s some contention in https://github.com/w3c/webdriver/pull/1445#discussion_r334529947 whether the script name should be part of the POST URL.

Marking this here so we remember to resolve the spec issue first.